brianlong/www.validators.app
# 183607614 software versions vue

#### What's this PR do?
- create software versions component

#### How should this be manually tested?
- run `rails r daemons/front_stats_update_daemon.rb`
- check bottom of the hme-new page for software versions

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/183607614)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
